cli/keyring_export.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func MakeKeyringExportCommand() *cobra.Command {
```

Comment:
question: What are your plans for testing all of this? Are you looking to just rely on manual testing?

Reply:
Wouldn't need to mock to test it I don't think - tests can handle things like password prompts without to much hassle.

I'm a bit nervous about not pushing for this (either here or in a follow up PR), as apart from the openAPI stuff it would be the only thing in the whole repo that really requires manual testing.

Reply:
I'll work on adding some tests. I think I have an idea on how to avoid the user input.

Reply:
I've added unit tests for all keyring commands.